utility.py

```diff
@@ -0,0 +1,9 @@
+MANAGER_ALIASES = {
+	"Joe Guidoboni" : "Poopoo Peepants",
+	"Brendan Shea" : "Real League Daddy",
+}
+
+def clean_name(name: str) -> str:
+    """Cleans up a manager's name str and fetches its alias, if present"""
+    cleaned = name.replace("  ", " ").strip().title()
+    return MANAGER_ALIASES[cleaned] if cleaned in MANAGER_ALIASES else cleaned
```

Comment:
I think you can simplify this last line with a little bit of Python dict fanciness

```
return MANAGER_ALIASES.get(cleaned, cleaned)
```

Reply:
good call, much cleaner. updating